src/layouts/default.html.eco

```diff
@@ -113,17 +113,36 @@
     <img src="<%= @site.assets %>/img/top.png" alt="<%= @getLang().goTop %>" width="64" height="64">
   </a> -->
 
-  <script type="text/javascript">
-    var _gaq = _gaq || [];
-    _gaq.push(['_setAccount', 'UA-4114546-34']);
-    _gaq.push(['_trackPageview']);
-
-    (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
-  </script>
+   <script type="text/javascript">
+      // Script
+      window.addEventListener('scroll',
+         ThrottleFixedMenu
+      );
+
+      function ThrottleFixedMenu(){
+         var TopElement = document.getElementById('menu').offsetTop;
+         var MainMenu   = document.getElementById("menu");
+         var TamScroll  = window.pageYOffset || document.documentElement.scrollTop;
+         var TamGlobal  = 562;
```

Comment:
It is menu initial time I store because the time is reset when it becomes fixed.